app/src/main/java/org/oppia/android/app/databinding/ViewBindingAdapters.java

```diff
@@ -21,6 +21,9 @@ public static void setLayoutHeight(@NonNull View view, float height) {
     view.setLayoutParams(layoutParams);
   }
 
+  /**
+   * BindingAdapter to set Rotation Animation and Rotation Angle.
+   */
```

Comment:
Nit:
We can write this in single line too. Right?
`/** BindingAdapter to set Rotation Animation and Rotation Angle. */`

Reply:
ah, missed it. I will make a consistency here in all.

Reply:
done